server/routes/admin.js

```diff
@@ -0,0 +1,6 @@
+const router = require('express').Router();
+const { signupAdmin } = require('../controllers/admins');
+
+router.post('/signupadmin', signupAdmin);
```

Comment:
You already used `/admin` in the router so this path will became `/admin/signup`
```suggestion
router.post('/signup', signupAdmin);
```